src/bid.rs

```diff
@@ -166,6 +169,47 @@ impl Bid {
         Ok(bid)
     }
 
+    /// Returns the `encrypted_data` field of the [Bid](self::Bid).
+    pub fn encrypted_data(&self) -> PoseidonCipher {
+        self.encrypted_data
+    }
+
+    /// Returns the `nonce` field of the [Bid](self::Bid).
+    pub fn nonce(&self) -> BlsScalar {
+        self.nonce
+    }
+
+    /// Returns the `hashed_secret` field of the [Bid](self::Bid).
+    pub fn hashed_secret(&self) -> BlsScalar {
+        self.hashed_secret
+    }
+
+    /// Returns the `commitment` field of the [Bid](self::Bid).
+    pub fn commitment(&self) -> JubJubAffine {
+        self.c
+    }
+
+    /// Returns the `eligibility` field of the [Bid](self::Bid).
+    pub fn eligibility(&self) -> u64 {
+        self.eligibility
+    }
+
+    /// Returns the `expiration` field of the [Bid](self::Bid).
+    pub fn expiration(&self) -> u64 {
+        self.expiration
+    }
+
+    /// Returns the `pos` field of the [Bid](self::Bid).
+    pub fn pos(&self) -> u64 {
+        self.pos
+    }
+
+    /// Returns a mutable ref pointing to the `pos` field of the
+    /// [Bid](self::Bid).
+    pub fn pos_mut(&mut self) -> &mut u64 {
```

Comment:
As mentioned in chat, according to https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#getter/setter-methods-[rfc-344] it should be called `set_pos`.